ui/cypress/e2e/tasks.test.ts

```diff
@@ -61,80 +61,69 @@ from(bucket: "${name}")
       .and('contain', taskName)
   })
 
-  it('can delete a task', () => {
-    cy.get('@org').then(({id}: Organization) => {
-      cy.get<string>('@token').then(token => {
-        cy.createTask(token, id)
-        cy.createTask(token, id)
-      })
-
-      cy.fixture('routes').then(({orgs}) => {
-        cy.visit(`${orgs}/${id}/tasks`)
-      })
-
-      cy.getByTestID('task-card').should('have.length', 2)
-
-      cy.getByTestID('task-card')
-        .first()
-        .trigger('mouseover')
-        .within(() => {
-          cy.getByTestID('context-delete-menu').click()
-          cy.getByTestID('context-delete-task').click()
+  describe('When tasks already exist', () => {
+    beforeEach(() => {
+      cy.get('@org').then(({id}: Organization) => {
+        cy.get<string>('@token').then(token => {
+          cy.createTask(token, id)
         })
-
-      cy.getByTestID('task-card').should('have.length', 1)
-    })
-  })
-
-  it('can disable a task', () => {
-    cy.get('@org').then(({id}: Organization) => {
-      cy.get<string>('@token').then(token => {
-        cy.createTask(token, id)
       })
     })
 
-    cy.getByTestID('task-card--slide-toggle').should('have.class', 'active')
+    it('can edit a task', () => {
+      // Disabling the test
+      cy.getByTestID('task-card--slide-toggle').should('have.class', 'active')
 
-    cy.getByTestID('task-card--slide-toggle').click()
+      cy.getByTestID('task-card--slide-toggle').click()
 
-    cy.getByTestID('task-card--slide-toggle').should('not.have.class', 'active')
-  })
+      cy.getByTestID('task-card--slide-toggle').should(
```

Comment:
linter might want this on 1 line

Reply:
this was produced by running `yarn prettier:fix` /shrug

Reply:
🤷‍♂